packages/sanity/src/core/form/inputs/PortableText/object/BlockObject.tsx

```diff
@@ -115,6 +115,7 @@ export function BlockObject(props: BlockObjectProps) {
         onClickingEdit={openItem}
         readOnly={readOnly}
         renderPreview={renderPreview}
+        isOpen={memberItem?.member.open}
```

Comment:
Even when the default is "undefined" it will still translate as false which is fine because this component will update once the member is finally opened / closed.

Reply:
You could do a `!!memberItem?.member.open` to keep it all `Boolean`.